### ISSUE TITLE
2FA and ESCROW 

### ISSUE THREAD

**Issue by GreYzZ** [ztizzlegaming/CSGOWinBig-SteamBot#70]
waylaidwanderer released his version of the Steambot.
Which has 2FA and ESCROW enabled.
Maybe @ztizzlegaming can add this to this CSGOWinBig-Steambot
or someone else who is used to do this.

**Comment by TheAnthonyNL** [comment]
Download that bot and then from csgowinbig steambot copy over the DepositTradeOfferUserHandler.cs  ;)

**Comment by TheAnthonyNL** [comment]
Oops also dont forget  to copy the sentryfiles and settings.json over.

**Comment by Stuffa1991** [comment]
I wish it was that simple @TheAnthonyNL :)

**Comment by GreYzZ** [comment]
@Stuffa1991 
isn't it?
Can you help compiling?

**Comment by Stuffa1991** [comment]
I tried what @TheAnthonyNL didnt work, ill see if i can figure something out later :)

**Comment by TheAnthonyNL** [comment]
Yeah i thought the same but im facing errors which im tryong to fix.

**Comment by GreYzZ** [comment]
@ztizzlegaming @TheAnthonyNL 
already a idea why these errors happens?

**Comment by TheAnthonyNL** [comment]
Well fixed my errors and its linked right now aswell 2fa but ... now awaiting on how it processes the trade offers...

**Comment by GreYzZ** [comment]
@TheAnthonyNL 
so you can upload the fixed bot?

**Comment by TheAnthonyNL** [comment]
Im writing a manual soon ... but first im gonna apply https://www.reddit.com/r/SteamBot/comments/3veuaq/how_to_know_if_the_sender_has_mobile_confirmation/
 to it.

**Comment by Xeroner** [comment]
@TheAnthonyNL When the it will be ready?

**Comment by TheAnthonyNL** [comment]
As soon steambot devs will update the tradeoffer class ;).

**Comment by Xeroner** [comment]
Cool :) @TheAnthonyNL We are waiting. BTW, can you reply for my topic?
https://github.com/ztizzlegaming/CSGOWinBig-SteamBot/issues/71

**Comment by Xeroner** [comment]
@TheAnthonyNL  ??

**Comment by TheAnthonyNL** [comment]
Ok, bot is done ( https://github.com/TheAnthonyNL/SteamBot ). I will create a guide for it on how to set it up so please wait before you do anything else you will screw it up.

**Comment by Xeroner** [comment]
Will this change block trade for my bot?

**Comment by TheAnthonyNL** [comment]
Block trade for what ?

**Comment by Xeroner** [comment]
For bot, if i will change something. Something like 7 days trade block or something

**Comment by TheAnthonyNL** [comment]
No BUT JUST WAIT OK, i will release a guide in 12~14 hours from now...

**Comment by Xeroner** [comment]
ok

**Comment by TheAnthonyNL** [comment]
The guide : https://github.com/TheAnthonyNL/SteamBot/wiki/How-to-use-this-SteamBot-for-CSGOWINBIG

**Comment by TheAnthonyNL** [comment]
If anything is not clear please contact me...

**Comment by Xeroner** [comment]
@TheAnthonyNL Everything cool but i can authorize the bot :((

[CSBOX.PL 2015-12-09 01:55:51] ERROR: Failed to generate 2FA code. Make sure you have linked the authenticator via SteamBot.
[CSBOX.PL 2015-12-09 01:55:52] ERROR: Login Error: AccountLoginDeniedNeedTwoFactor
[CSBOX.PL 2015-12-09 01:55:52] ERROR: Failed to generate 2FA code. Make sure you have linked the authenticator via SteamBot.
[CSBOX.PL 2015-12-09 01:56:09] ERROR: Login Error: AccountLoginDeniedNeedTwoFactor

**Comment by Stuffa1991** [comment]
Thanks for the fix @TheAnthonyNL :)

**Comment by TheRod1990** [comment]
Thanks @TheAnthonyNL !!!
